force-app/main/default/classes/Homework.cls

```diff
@@ -1,100 +1,137 @@
 public with sharing class Homework {
     // Declare a variable to hold the postcode of Ginetta House - as
     // the office cannot move, this should be immutable
-
+    public static final String bgPostCode = 'CM8 1BE';
```

Comment:
Good use of static to ensure we only have one literal regardless of how many instances of Homework there are.